students/philip_behrend/lesson03/mailroom.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Sep  9 22:18:06 2019
+
+@author: Philip Behrend
+"""
+import numpy as np
+
+class Donors:
+    def __init__(self,donor,amt_list):
+        self.donor = donor
+        self.amt_list = amt_list
+
+donor_list = Donors(['Marge','Harold','Henry','Myrtle','Mitchell'],
+                    [[50,40],[100,1000,10000],
+                     [2],[5,0.5,.05],[3,6,9]])
+
+def send_thanks(donor_list):
+    name_response = input("Type full name: ")
+    while True:
+        if name_response == "list":
+            print(donor_list.donor)
+        elif name_response in donor_list.donor:
+            donor_index = donor_list.donor.index(name_response)
+            donation_response = round(float(input("Type donation amount: ")),2)
+            assert donation_response >= 0
+            donor_list.amt_list[donor_index].append(donation_response)      
+            break
+        else:
+            donor_list.donor.append(name_response)
+            donation_response = round(float(input("Type donation amount: ")),2)
+            assert donation_response >= 0
+            donor_list.amt_list.append([donation_response])
+            break
```

Comment:
above two blocks have quite a bit of duplicated code - can some of this be combined?